docs/migration/00b_Electronic_structure_with_v0.5.ipynb

```diff
@@ -0,0 +1,729 @@
+    "\n",
+    "| New component | Legacy location |\n",
+    "|:--------------|:----------------|\n",
+    "| `BaseProblem` | `qiskit_nature.problems.second_quantization.BaseProblem` |\n",
+    "| `EigenstateResult` | `qiskit_nature.results.EigenstateResult` |\n",
+    "| `PropertiesContainer` | similar to `qiskit_nature.properties.GroupedProperty` |\n",
+    "| `ElectronicBasis` | `qiskit_nature.properties.second_quantization.electronic.bases.ElectronicBasis` |\n",
+    "| `ElectronicStructureProblem` | `qiskit_nature.problems.second_quantization.electronic.ElectronicStructureProblem` |\n",
+    "| `ElectronicPropertiesContainer` | _did not exist yet_ |\n",
+    "| `ElectronicStructureResult` | `qiskit_nature.results.ElectronicStructureResult` |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd66694c",
+   "metadata": {},
+   "source": [
+    "### Previously"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "24b9c43d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_nature.drivers.second_quantization import PySCFDriver\n",
+    "from qiskit_nature.problems.second_quantization.electronic import ElectronicStructureProblem\n",
+    "from qiskit_nature.transformers.second_quantization.electronic import FreezeCoreTransformer\n",
+    "\n",
+    "driver = PySCFDriver()\n",
+    "transformer = FreezeCoreTransformer()\n",
+    "problem = ElectronicStructureProblem(driver, transformers=[transformer])\n",
+    "\n",
+    "# we trigger driver.run() implicitly like so:\n",
+    "second_q_ops = problem.second_q_ops()\n",
+    "\n",
+    "hamiltonian_op = second_q_ops.pop(\"ElectronicEnergy\")\n",
+    "aux_ops = second_q_ops"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73fe4aa5",
+   "metadata": {},
+   "source": [
+    "### New"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "d6a36494",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_nature.second_q.drivers import PySCFDriver\n",
+    "from qiskit_nature.second_q.transformers import FreezeCoreTransformer\n",
+    "\n",
+    "driver = PySCFDriver()\n",
+    "\n",
+    "problem = driver.run()\n",
+    "\n",
+    "transformer = FreezeCoreTransformer()\n",
+    "\n",
+    "problem = transformer.transform(problem)\n",
+    "\n",
+    "hamiltonian_op, aux_ops = problem.second_q_ops()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57d08f45",
+   "metadata": {},
+   "source": [
+    "For more information on the new and improved `ElectronicStructureProblem`, please refer to the [electronic structure tutorial](../tutorials/01_electronic_structure.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e041f05",
+   "metadata": {},
+   "source": [
+    "## `qiskit_nature.properties`\n",
+    "\n",
+    "The properties module has been refactored and split into multiple locations. In this section, we will only be focusing on its _electronic_ components."
```

Comment:
For people that want to inject integrals they got from elsewhere - or get the integrals from a driver do/should we have anything for this. I know we get users that want to do either and these aspects are changed again right though these are more advanced users so maybe just seeing where/how now in some tutorial or other would suffice.

Reply:
Yes, I was thinking that the tutorial should suffice for this aspect. I can add a specific mention of this in the migration guide though :+1: